packages/ui-extensions/src/surfaces/checkout/components/Button/examples/basic-button.example.html

```diff
@@ -1 +1,4 @@
-<s-button onclick="console.log('onClick event')">Pay now</s-button>
+<s-button onClick="console.log('onClick event')">Cancel</s-button>
+<s-button variant="primary" onClick="console.log('onClick event')">
```

Comment:
Lets keep things simple. No need to show events. Also match Admin.
```suggestion
<s-button variant="secondary">Cancel</s-button>
<s-button variant="primary">
```